plugins/drivers/testutils/exec_testing.go

```diff
@@ -192,9 +194,35 @@ func TestExecFSIsolation(t *testing.T, driver *DriverHarness, taskID string) {
 			false, "")
 		require.Zero(t, r.exitCode)
 
-		if !strings.Contains(r.stdout, ":freezer:/nomad") && !strings.Contains(r.stdout, "freezer:/docker") {
-			require.Fail(t, "unexpected freezer cgroup", "expected freezer to be /nomad/ or /docker/, but found:\n%s", r.stdout)
+		if !cgutil.UseV2 {
+			acceptable := []string{
+				":freezer:/nomad", ":freezer:/docker",
+			}
+			if testutil.IsCI() {
+				// github actions freezer cgroup
+				acceptable = append(acceptable, ":freezer:/actions_job")
```

Comment:
Oof, I imagine you discovered this experimentally and it wasn't documented? Probably nothing we can do about it but some day that's going to change out from under us and break a bunch of tests. So at least you've documented it! 😀

Reply:
Heh yeah, it's really got me ~concerned~ curious how GHA works under the hood ...